examples/k8s-overlay-ingress/ReadMe.md

```diff
@@ -0,0 +1,16 @@
+This example shows how to use `ytt` overlaying in regards to accomplish the modification of X Kubernetes Ingress object.
+
+Specifically the `apiVersion` is bumped to **networking.k8s.io/v1** from **networking.k8s.io/v1beta1**. As a consequence this requires us to
+`ytt overlay` on **backend** and **paths** on the **Kubernetes Ingress Object**. In order for these to match the `apiVersion` that we bump to.
+
+Specifically the method is used on Falco Helm chart having issues with the version tag that K3s uses. Resulting in that chart using the **v1beta1** of the **networking** API.
```

Comment:
The readme explains _what_ is being modified, but not _why_ `apiVersion`, and other places need to be modified. Can you give a brief description of what the problem is that this example solves? Can you give some detail on what environment the problem shows up?